From: Christophe Giboudeaux <cgiboudeaux@gmx.com>
Date: Mon, 02 Apr 2012 12:38:56 +0000
Subject: Build with Clang.
X-Git-Url: http://quickgit.kde.org/?p=akonadi-google.git&amp;a=commitdiff&amp;h=3e0a93e1b24cd7b6e394cf76d153c428246f9fa9
---
Build with Clang.
Exception is ambiguous. It's defined in both common.h and kdepimlibs/akonadi/exception.h
---


--- ./calendar/calendarresource.cpp
+++ ./calendar/calendarresource.cpp
@@ -141,7 +141,7 @@ void CalendarResource::configure(WId win
         try {
             Auth *auth = Auth::instance();
             m_account = auth->getAccount(Settings::self()->account());
-        } catch (Exception::BaseException &e) {
+        } catch (KGoogle::Exception::BaseException &e) {
             emit status(Broken, e.what());
             delete settingsDialog;
             return;

--- ./contacts/contactsresource.cpp
+++ ./contacts/contactsresource.cpp
@@ -87,7 +87,7 @@ ContactsResource::ContactsResource(const
     if (!Settings::self()->account().isEmpty()) {
         try {
             m_account = auth->getAccount(Settings::self()->account());
-        } catch (Exception::BaseException &e) {
+        } catch (KGoogle::Exception::BaseException &e) {
             emit status(Broken, e.what());
             return;
         }
@@ -137,7 +137,7 @@ void ContactsResource::configure(WId win
         try {
             Auth *auth = Auth::instance();
             m_account = auth->getAccount(Settings::self()->account());
-        } catch (Exception::BaseException &e) {
+        } catch (KGoogle::Exception::BaseException &e) {
             emit status(Broken, e.what());
             delete settingsDialog;
             return;

